bad_code.java

```diff
@@ -0,0 +1,54 @@
+public class BadCodeExample {
+    
+    public static void main(String[] args) {
+        ChildClass child = new ChildClass();
+        child.doSomething();
+        child.doSomethingElse();
+    }
+}
+
+class ParentClass {
+    public int someNumber = 10;
```

Comment:
Consider using access modifiers for fields and methods in `ParentClass` and `ChildClass` to encapsulate their visibility and prevent unintended modifications. [important]